src/misc/host_locale_win32.cpp

```diff
@@ -31,6 +31,319 @@
 
 CHECK_NARROWING();
 
+// Mapping from modern Windows to DOS keyboard layout. Developed using
```

Comment:
"layouts"

Reply:
OK, changed - also in POSIX counterpart tables.